### PR TITLE
Added a `.hint()` method to allow hints in the quiz

### PR DESCRIPTION
**Pull request by rw251**
I've made a default hint which reminds people about `show()`. I realise this method is currently still called `debug()` but I doubt this will affect anyone before the debug->show change is also made.

### CODE REVIEW

File: ehrql/quiz.py
Comment (inglesp) [inline ehrql/quiz.py:335]: It'd be more idiomatic to use `textwrap.dedent` for this.

**Review by inglesp — APPROVED** [review]
Good stuff.  What's the plan for adding some real hints?  Once we've done that we'll also need to document this.

**Comment by rw251** [comment]
> Good stuff. What's the plan for adding some real hints? Once we've done that we'll also need to document this.

I've already got some hints ready to go. I was waiting for this to merge so that the hint method is actually available, and for debug to be changed to show. Therefore once this is merged I'll tidy up my hints and raise a PR.

**Pull request merged**
